src/feature/People/PeopleList/List/index.js

```diff
@@ -33,11 +34,13 @@ const PopularPeopleList = () => {
         {people.map((person) => (
           <Item key={person.id}>
             <Tile>
-              <Poster
-                src={`https://image.tmdb.org/t/p/w500/${person.profile_path}`}
-                alt={person.name}
-              />
-              <Title>{person.name}</Title>
+              <Link to={`/person/${person.id}`}>
```

Comment:
Wouldn't it be better to make <Link> above <Tile> to wrap poster and name? What I mean is when you see actor and his picture you can click on it or on his name, wherever you want at tile and it will redirect you to PeopleDetails?

Reply:
I think I don't understand.. Now (in this pull)you can click on whole Tile and this Tile will redirect you to the PesonDetails page with the logic related to current actor. Or you mean

<Item key={person.id}>
  <Link to={`/person/${person.id}`}>
   <Tile>             
    <Poster
        src={`https://image.tmdb.org/t/p/w500/${person.profile_path}`}
        alt={person.name}
         />
    <Title>{person.name}</Title>  
  </Tile>
 </Link>
</Item>
?

Reply:
I made another commit in this pull https://github.com/AdrianJar/movie-project/pull/40/commits/9ab3ca982ecfda08b93717488daed1de4af2f88b